### ISSUE TITLE
Finish 1.0 release

### ISSUE THREAD

**Issue by lifflander** [DARMA-tasking/LB-analysis-framework#440]
- Create release on Github
- Create package

**Comment by ppebay** [comment]
I think this issue was fully resolved by #448 , can we close it @lifflander @cwschilly @pierrepebay ?

**Comment by cwschilly** [comment]
Yes @ppebay , this has been resolved. However, given the issue in #453 (fixed in PR #454) we may want to consider another patch release 1.0.1

**Comment by ppebay** [comment]
Alright so let's close this issue and create a follow-on for 1.0.1
